GitFyle.Core.Api/Brokers/Storages/StorageBroker.Sources.Configuration.cs

```diff
@@ -0,0 +1,26 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using GitFyle.Core.Api.Models.Foundations.Sources;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace GitFyle.Core.Api.Brokers.Storages
+{
+    internal partial class StorageBroker
+    {
+        void AddSourceConfigurations(EntityTypeBuilder<Source> builder)
```

Comment:
Add this in StorageBroker.cs and also add a migration for the unique constraint and length restrictions

```cs
        protected override void OnModelCreating(ModelBuilder modelBuilder)
        {
            AddSourceConfigurations(modelBuilder.Entity<Source>());
        }
```

Reply:
currently, the OnModelCreating method is in the Contribution branch I guess. Adding it here will result In merge conflicts while waiting for it to be merged. and then we can add these configurations here.

Reply:
You are OK to add it here.  We will deal with potential conflicts when the other branches are merged.  We don't need to keep track of what is going on in other branches.  You only make sure that your branch is up to date with `main` any changes in other PR's is out of scope for this bit of work.